UI/RemoteISOScreen.cpp

```diff
@@ -268,19 +275,34 @@ static bool LoadGameList(const std::string &host, int port, std::vector<std::str
 	std::vector<std::string> items;
 	result.TakeAll(&listing);
 
-	SplitString(listing, '\n', items);
-	for (const std::string &item : items) {
-		if (!endsWithNoCase(item, ".cso") && !endsWithNoCase(item, ".iso") && !endsWithNoCase(item, ".pbp")) {
-			continue;
+	if (startsWith(responseHeaders[0],"Server: SuperDuperServer")) {
```

Comment:
Done, but I think we should leave SuperDuperServer in there for now as a check.  So it will work correctly if a v1.3 is the server for a v1.4+ client.